app.go

```diff
@@ -445,9 +450,15 @@ func (app *App) Stop(ctx context.Context) error {
 // application. Applications listen for the SIGINT and SIGTERM signals; during
 // development, users can send the application SIGTERM by pressing Ctrl-C in
 // the same terminal as the running process.
+// Alternatively, a signal can be broadcast to all done channels manually by
+// using the Shutdown functionality (see the Shutdowner documentation for details)
```

Comment:
nit: please add a period, exclamation point, or terminal punctuation of your choice at the end of this sentence.